core/services/relay/evm/mercury/wsrpc/cache/cache_set.go

```diff
@@ -0,0 +1,118 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+// NOTE: CacheSet holds a set of mercury caches keyed by server URL
+
+type CacheSet interface {
```

Comment:
Any reason not to have this be the doc? It is dangling otherwise (from the tooling's perspective)
```suggestion
// CacheSet holds a set of mercury caches keyed by server URL
type CacheSet interface {
```

Reply:
Actually, should this mention mercury at all?